src/systemd/tmpfiles.d/cockpit-ws.conf.in

```diff
@@ -1,3 +1,4 @@
+z @libexecdir@/cockpit-session - - cockpit-wsinstance -
```

Comment:
This is going to be `/usr/libexec/...` right? This will fail on all Atomic desktops, CoreoS, etc.

Reply:
Shouldn't -- *if* you install an rpm on CoreOS etc., then /usr needs to be writable anyway (or some overlay, etc.). Normally you'd get it from some image build process or through an OCI container build etc. -- either way, this should be a no-op on read-only /usr as the files/permissions should already be set correctly.

I triggered a fedora-coreos test run to make sure. (I didn't trigger all tests yet, as this is still draft).

Reply:
Hum, I don't understand when this is supposed to run then and why if it's not at runtime, when /usr is RO on the nodes.

Reply:
It runs at two times:
 - At boot. This is mostly relevant for upstream `sudo make install` situations, or when there's some kind of "full /etc" reset. That part wouldn't work for read-only /usr/ of course.
 - At package installation time (from `%post` rpm script). That's when rpm just unpacked the files to /usr, so at that time /usr necessarily *has* to be writable (by whatever means).

Reply:
OK, this should work.